common/src/main/webapp/sass/solar-system.scss

```diff
@@ -1,98 +0,0 @@
-/* KEYFRAMES */
-
-@keyframes spin {
-  from {
-    transform: rotate(0);
-  }
-
-  to{
-    transform: rotate(360deg);
-  }
-}
-
-/* GRID STYLING */
-
-:root {
```

Comment:
Have we lost the solar system spinner? 😢

Reply:
Yes, it is now on lucuma-ui